static/app/views/issueDetails/eventNavigation.tsx

```diff
@@ -0,0 +1,233 @@
+    condition: (event: Event) => event.entries.find(entry => entry.type === 'stacktrace'),
+  },
+  {
+    section: 'exception',
+    label: t('Exception'),
+    condition: (event: Event) => event.entries.find(entry => entry.type === 'exception'),
+  },
+  {
+    section: 'breadcrumbs',
+    label: t('Breadcrumbs'),
+    condition: (event: Event) =>
+      event.entries.find(entry => entry.type === 'breadcrumbs'),
+  },
+  {section: 'tags', label: t('Tags'), condition: (event: Event) => event.tags.length > 0},
+  {section: 'context', label: t('Context'), condition: (event: Event) => event.context},
+  {
+    section: 'user-feedback',
+    label: t('User Feedback'),
+    condition: (event: Event) => event.userReport,
+  },
+  {
+    section: 'replay',
+    label: t('Replay'),
+    condition: (event: Event) => getReplayIdFromEvent(event),
+  },
+];
+
+export default function EventNavigation({event, group}: EventNavigationProps) {
+  const location = useLocation();
+  const navigate = useNavigate();
+  const organization = useOrganization();
+
+  const hasPreviousEvent = defined(event.previousEventID);
+  const hasNextEvent = defined(event.nextEventID);
+
+  const baseEventsPath = `/organizations/${organization.slug}/issues/${group.id}/events/`;
+
+  const jumpToSections = eventDataSections.filter(eventSection =>
+    eventSection.condition(event)
+  );
+
+  return (
+    <div>
+      <EventNavigationWrapper>
+        <Tabs>
+          <TabList hideBorder>
+            {Object.keys(EventNavLabels).map(label => {
+              return (
+                <TabList.Item
+                  to={{
+                    pathname: normalizeUrl(baseEventsPath + label + '/'),
+                    query: {...location.query, referrer: `${label}-event`},
+                  }}
+                  key={label}
+                >
+                  {EventNavLabels[label]}
+                </TabList.Item>
+              );
+            })}
+          </TabList>
+        </Tabs>
+        <NavigationWrapper>
+          <Navigation>
+            <LinkButton
+              title={'Previous Event'}
+              aria-label="Previous Event"
+              borderless
+              size="sm"
+              icon={<IconChevron direction="left" />}
+              disabled={!hasPreviousEvent}
+              to={{
+                pathname: `${baseEventsPath}${event.previousEventID}/`,
+                query: {...location.query, referrer: 'previous-event'},
+              }}
+            />
+            <LinkButton
+              title={'Next Event'}
+              aria-label="Next Event"
+              borderless
+              size="sm"
+              icon={<IconChevron direction="right" />}
+              disabled={!hasNextEvent}
+              to={{
+                pathname: `${baseEventsPath}${event.nextEventID}/`,
+                query: {...location.query, referrer: 'next-event'},
+              }}
+            />
+          </Navigation>
+          <StyledButton
```

Comment:
Is there a reason why this isn't a `LinkButton` like the others?

Reply:
no good reason, had it before i switched over to using LinkButton but didn't update this one